addons/udes_stock/models/stock_picking_type.py

```diff
@@ -43,6 +43,20 @@ class StockPickingType(models.Model):
         the system.
         """,
     )
+    # Enable user process partial
+    u_user_process_partial = fields.Boolean(
```

Comment:
Personal nit: This name confuses me. Is it possible to have something else like
u_multi_users_enabled or something, that way I know that many can work on it at the same time. Currently in the UI i always have to look in the code, because i always think this means under receive something, or do part of it. 
But the teams leads may reject this, just a personal preference.

Reply:
Will wait for a TL suggestion, but i agree with you :)

Reply:
I agree personally, it's not abundantly clear what this flag is doing. Happy to rename the field to something clear and update the help text. Also update the README so we know what this field used to be called. `u_multi_users_enabled` is a good suggestion.